apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/ExplanationOfBenefitE2E.java

```diff
@@ -695,15 +694,13 @@ public void testEobByPatientIdWithLastUpdated() {
   /**
    * Verifies that {@link ExplanationOfBenefitResourceProvider#findByPatient} works as with a
    * lastUpdated parameter after yesterday and pagination links work and contain lastUpdated.
-   *
-   * <p>FIXME: Fix up like v2, should work now
```

Comment:
I believe the reason why it passes. Is the because the resourcesSubList eventually equals the resources size. It just sends it in via the pagination logic as expected. I'll change to resourcesSublist for uniformity with the TransformerUtilsV2. But both will pass.

Reply:
Oh that spot where its using sublist at the bottom is just for the metadata lastUpdated time, which I don't even think is tested. I guess thats why it doesnt affect the test.

Reply:
Correct.